src/test/java/com/disney/pg2k4j/KinesisReceivesPostgresChangesIT.java

```diff
@@ -0,0 +1,111 @@
+package com.disney.pg2k4j;
+
+import com.amazonaws.SDKGlobalConfiguration;
+import com.amazonaws.services.kinesis.model.GetRecordsResult;
+import com.amazonaws.services.kinesis.model.Record;
+import com.disney.pg2k4j.containers.KinesisLocalStack;
+import com.disney.pg2k4j.containers.Postgres;
+import com.disney.pg2k4j.models.Change;
+import com.disney.pg2k4j.models.InsertChange;
+import com.disney.pg2k4j.models.SlotMessage;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Network;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+public class KinesisReceivesPostgresChangesIT {
```

Comment:
Is there any reason for not having the integration tests in a separate (`it`) package so they run separately? Generally having integration tests run with the unit tests can cause a number of issues:
* It skews the unit test coverage
* They often take longer to run 
* they often require more dependencies, both library and infrastructure

Reply:
Unit tests run as part of the surefire plugin and run files named like "Test"
https://github.com/disneystreaming/pg2k4j/blob/integrationtests/pom.xml#L141-L146
Integration tests run as part of the failsafe plugin and run files named like "IT"
https://github.com/disneystreaming/pg2k4j/blob/integrationtests/pom.xml#L155

I can split these up into different packages to make that distinction more clear.

Reply:
Tests can be run using the Maven Failsafe Plugin, which by default excludes these [integration tests by convention](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html).

In the past, we've enjoyed using this pattern of executing integration tests within a JUnit runnable, which not only gives us a nice way of debugging, but also a friendly way for maven to execute these tests as part of a heavier-weight test phase.

We certainly want to make the distinction between the unit test phase and the integration test phase, which the failsafe plugin show allow for. We should also be able to articulate which set of tests should be run as part of a full `mvn install`.

I think @markglh is right in that we need to be careful in excluding the results from the integration tests in our code coverage results, and I'm not sure that happens by default. However, the heavier nature of these tests are a tradeoff we're willing to make, as the overall correctness of this framework is more important (from our perspective) than a slower build.

Reply:
Makes sense :)
No need to split them up. I agree regarding slower build given the size of the project, provided they run as separate phases it's all good!